specs/centos_ci.adoc

```diff
@@ -0,0 +1,135 @@
+// vim: tw=79
+
+= CentOS CI
+
+We need to make sure that new code doesn't break existing codebase and check
+that new features works as early as possible. This can be done by running
+acceptance integration tests automatically, using CentOS CI infrastructure.
+
+== Problem description
+
+CentOS CI
+will be configured to launch acceptance integration test run for every new
+nightly rpm build from tendrl nightly copr project. As set of implemented
```

Comment:
This is being discussed under https://github.com/Tendrl/specifications/issues/53 as well. As I noted there in https://github.com/Tendrl/specifications/issues/53#issuecomment-268336489 , I worry about number of concurrent, independent builds (one for each specification under development).